sdk/src/main/java/com/horizen/account/state/ExecutionSucceeded.java

```diff
@@ -0,0 +1,38 @@
+package com.horizen.account.state;
+
+import java.math.BigInteger;
+
+
+// Message was executed as expected.
+public class ExecutionSucceeded implements ExecutionResult {
+    private final BigInteger gasUsed; // Total used gas during Message processing
+    private final byte[] returnData; // Returned data from invocation(function result or data supplied with revert opcode)
```

Comment:
Where do we put the logs? Also here?

Reply:
I think, that logs are add to the View directly while processing message. Similar to the Geth implementation.
So, inside StateDB we cache the logs linked to tx id.